2/Dockerfile.rhel7

```diff
@@ -51,9 +51,10 @@ ADD ./contrib/s2i /usr/libexec/s2i
 # and its presence in the rhel image is confusing.
 RUN rm /opt/openshift/base-plugins.txt && \ 
     mkdir -p /opt/openshift/plugins && \
-    # NOTE: When adding new Jenkins plugin, you have to create the symlink for the
-    # HPI file created by rpm to /opt/openshift/plugins folder.
-    for FILENAME in /usr/lib/jenkins/*hpi ; do ln -s $FILENAME /opt/openshift/plugins/`basename $FILENAME .hpi`.jpi; done &&\
+    # for backward compatibility with pre-3.6 installs leveraging a PV, where rpm installs went to /usr/lib64/jenkins, we are
+    # establishing a symbolic link for that guy as well, so that existing plugins in JENKINS_HOME/plugins pointing to /usr/lib64/jenkins
+    # will subsequently get redirected to /usr/lib/jenkins; it is confirmed that the 3.7 jenkins RHEL images do *NOT* have a /usr/lib64/jenkins path
+    ln -s /usr/lib/jenkins /usr/lib64/jenkins && \
```

Comment:
I still think this needs to happen before we yum install anything, to avoid the scenario where a package creates this dir(/usr/lib64/jenkins) and then we can't create the symlink.

Reply:
ok i misinterpreted the earlier comment then (thinking it only pertained to this run block)

moving above